XRTK-Core/Packages/com.xrtk.core/Runtime/Services/BoundarySystem/MixedRealityBoundarySystem.cs

```diff
@@ -61,6 +67,30 @@ public override void Initialize()
             }
 
             boundaryEventData = new BoundaryEventData(EventSystem.current);
+
+            if (inputSubsystem == null)
```

Comment:
I need to go back and make sure we're handling the lifetime of this like in the other places

Reply:
Tbh I'd rather lose the dependency on the xr subsystem. We'll probably need a way to distinguish the tracking tracking origin mode as well, but tbh, does it even matter? We're already relocalizing all the relative input in each controller data provider. I also think the boundary system should have it's own data provider configurations as well.